cmake-tests/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ if(NOT CMAKE_CXX_COMPILER MATCHES ".*hcc")
 endif()
 
 add_executable(cmake-test cmake-test.cpp)
+add_dependencies(cmake-test hc_am mcwamp_hsa mcwamp)
```

Comment:
This should happen under `if(TARGET hccrt)` below, because this is ran both in build and standalone(in order to test `find_package(hcc)`).

Also `hc_am` is already added as a dependency through `target_link_libraries`.